### PR TITLE
more styling ref

### PR DESCRIPTION
**Pull request by oerkperl**

### CODE REVIEW

File: app/components/coin/LinksList.tsx [hunk 1]
```diff
@@ -1,33 +1,120 @@
...
+//                 <p>{trimLink(link)}</p>
+//               </div>
+//             </Link>
+//           ) : (
+//             ""
+//           )
+//         )}
+//     </div>
+//   );
+// };
```
Comment (yoloOnTheBattlefield) [inline app/components/coin/LinksList.tsx]: delete these lines

File: app/components/coin/LinksList.tsx [hunk 2]
```diff
@@ -1,33 +1,120 @@
...
+        className=" relative overflow-hidden w-full bg-white dark:bg-transparent rounded"
+        style={{ width, height }}
+      >
+        <div
+          ref={containerRef}
+          className="flex gap-2 overflow-x-hidden  px-8 relative max-w-full"
+          style={{ width: "fit-content" }}
+        >
+          {links &&
+            links.map((link) =>
```
Comment (yoloOnTheBattlefield) [inline app/components/coin/LinksList.tsx]: filter before you map

File: app/components/portfolio/MyAsset.tsx [hunk 3]
```diff
@@ -26,15 +28,26 @@ export const MyAsset: React.FC<{
...
+            >
+              {refeshIcon}
+            </button>
+          )}
+          {!hasError && (
+            <button
+              className="px-2 rounded-md hover:bg-pink-600 hover:text-white"
+              onClick={() => {
+                removeAsset(asset?.id);
+              }}
```
Comment (yoloOnTheBattlefield) [inline app/components/portfolio/MyAsset.tsx]: no need for curly brackets here

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

**Review by oerkperl — COMMENTED** [review]
sorted.

**Pull request merged**